### ISSUE TITLE
About num_workers

### ISSUE THREAD

**Issue by 1998CHC** [4DVLab/DexGrasp-Anything#23]
Thank you for your excellent work. 
May I ask whether **num_workers** in **grasp_gen_ur.yaml** can only be set to 0? I tried increasing it but got an error.

**Comment by ymzhong66** [comment]
Yes, I always set it to 0. When I try a larger value I get errors as well. I’ve attempted to fix it but haven’t figured out where the problem is yet.

**Comment by 1998CHC** [comment]
In the dataloader, simply add .detach() after mdata_qpos, i.e.:

`mdata_qpos = torch.cat([global_trans, joint_angle], dim=0).requires_grad_(True).detach()`

However, increasing num_workers has limited impact on training speed, and setting num_workers to 2 seems to be the fastest.
